static/js/vue-cdr-access/src/components/collectionBrowse.vue

```diff
@@ -8,7 +8,7 @@ Top level component used for the collection browse page
 </template>
 
 <script>
-    import listDisplay from "./listDisplay";
+    import listDisplay from "./listDisplay.vue";
```

Comment:
why do some of these imports need to be switch to `'@/components/browseDisplay.vue';` form, but others can be left like this?

Reply:
I think I meant to switch them all. Sometimes files couldn't be found if they weren't using the @/components syntax.